charms/kserve-controller/requirements-lint.txt

```diff
@@ -12,7 +12,7 @@ click==8.1.3
     #   black
 codespell==2.2.4
     # via -r requirements-lint.in
-flake8==6.0.0
+flake8==5.0.4
```

Comment:
Because flake6 [requires python >= 3.8.1](https://flake8.pycqa.org/en/latest/release-notes/6.0.0.html#features) and this list of requirements is being generated using [python 3.8.0](https://github.com/canonical/kserve-operators/blob/main/renovate.json#L7)